dist/Safe/t/safetie.t

```diff
@@ -0,0 +1,23 @@
+#!perl
+
+BEGIN {
+    require Config;
+    import Config;
+    if ($Config{'extensions'} !~ /\bOpcode\b/) {
+	print "1..0\n";
+	exit 0;
+    }
+}
+
+use strict;
+use warnings;
+use Test::More;
+use Safe;
+plan(tests => 1);
+
+my $c = new Safe;
+$c->deny_only();
+
+# Tie::Scalar has no "import" method, we test that it can still be used
+# within a safe.
+ok($c->reval("use Tie::Scalar; 1"));
```

Comment:
Do we really need to create a new test file for a single unit test of this functionality?  I recommend seeing whether this could go into one of the `t/safe?.t` files.

Reply:
Done. `t/safeload.t` seemed to be the most appropriate for this.